nncf/experimental/torch/fx/quantization/quantize_model.py

```diff
@@ -135,25 +135,25 @@ def compress_weights_impl(
     """
     Implementation of the `compress_weights()` method for the Torch Fx backend.
     """
-
-    compression_algorithm = WeightCompression(
-        mode,
-        ratio,
-        group_size,
-        ignored_scope,
-        all_layers,
-        sensitivity_metric,
-        awq,
-        subset_size,
-        scale_estimation,
-        gptq,
-        lora_correction,
-        backup_mode,
-        advanced_parameters,
-    )
-    graph = NNCFGraphFactory.create(model)
-    compressed_model = compression_algorithm.apply(model, graph, dataset=dataset)
-    compressed_model = GraphModule(compressed_model, compressed_model.graph)
-    compressed_model = _disallow_eval_train(compressed_model)
+    with torch.no_grad():
```

Comment:
What about `with torch.inference_mode():`? It might be even faster. 
https://pytorch.org/docs/stable/generated/torch.autograd.grad_mode.inference_mode.html
> Code run under this mode gets better performance by disabling view tracking and version counter bumps.

Reply:
done